composer.json

```diff
@@ -24,7 +24,9 @@
     },
     "require-dev": {
         "league/oauth2-facebook": "^1.1",
-        "symfony/security-guard": "^2.8|^3.0"
+        "symfony/security-guard": "^2.8|^3.0",
+        "phpunit/phpunit": "^4.8",
```

Comment:
I think we should allow 5.x version of PHPUnit in this case too with `^4.8|^5.0`, right?